src/appengine/libs/issue_management/jira/__init__.py

```diff
@@ -135,6 +135,14 @@ def save(self):
 
     self.itm.save(self)
 
+  def actions(self):
+    # To prevent abstract exception.
+    return
```

Comment:
probably just "pass".

Reply:
pass was also giving linting issues, any other alternative? 
Also, I just removed a pass statement in another function [here](https://github.com/google/clusterfuzz/pull/1832/files#diff-c61315aee0223e7d4d4767c339334fb7L34)

Reply:
Added, please check.